hi-ml/src/health_ml/utils/config_loader.py

```diff
@@ -0,0 +1,139 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from __future__ import annotations
+
+import importlib
+import inspect
+import logging
+from importlib.util import find_spec
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import param
+from importlib._bootstrap import ModuleSpec
+
+# from health_ml.deep_learning_config import DeepLearningConfig
+from health_ml.utils.common_utils import path_to_namespace
+from health_ml.utils.generic_parsing import GenericConfig
+
+
+class ModelConfigLoader(GenericConfig):
+    """
+    Helper class to manage model config loading
+    """
+    model_configs_namespace: Optional[str] = param.String(default=None,
+                                                          doc="Non-default namespace to search for model configs")
+
```

Comment:
At the moment, we require a namespace and a model config name, provided in two separate commandline args. It could be helpful to also allow fully qualified model names, like "ML.configs.FooModel"

Reply:
In addition, this could help to speed up model loading